packages/ngx-flicking/projects/ngx-flicking/src/lib/ngx-flicking.component.ts

```diff
@@ -121,7 +130,8 @@ export class NgxFlickingComponent implements OnInit, AfterViewInit, OnDestroy, O
       this.flicking.on(eventName, e => {
         e.currentTarget = this;
 
-        this.handleEvents.emit(e); // type is camelCase
+        // Style guide: Event - https://angular.io/guide/styleguide#dont-prefix-output-properties
+        this[eventName].emit(e);
```

Comment:
@jongmoon 

It will likely be necessary to deal with events that are added later.

Reply:
@daybrush Right! 👍